packages/babel-plugin-rn-stylename-to-style/__tests__/index.spec.js

```diff
@@ -274,6 +274,23 @@ pluginTester({
         return render()
       }
     `,
+    'dynamic part attribute. Array and Object. Multiple keys.': /* js */`
```

Comment:
Please ignore this file change. It is not related to this PR. Just an extra test for the CSS parts compilation which I had lying around in my master not commited.